.github/workflows/validate.yaml

```diff
@@ -22,6 +23,10 @@ jobs:
           - dashboard/service
           - dashboard/job
           - dashboard/cloudevent-receiver
+          - prober
```

Comment:
Can we make this automatic instead of relying on reviewers to spot it missing?

Reply:
Yup, it's automatic in mono.  Nghia said he's gonna port it here.